buildSrc/src/main/java/eu/xenit/dynamicextensionsalfresco/Versions.java

```diff
@@ -24,6 +24,13 @@
         put("60", "5.0.4.RELEASE");
     }};
 
+    public static final Map<String, String> geminiVersions = new HashMap<String, String>() {{
+        put("50", "1.0.2.RELEASE");
```

Comment:
Is this still required? The version will be overwritten by the resolution strategy anyway?